src/domain.module.ts

```diff
@@ -89,9 +91,11 @@ import { BalancesApiModule } from '@/datasources/balances-api/balances-api.modul
     DelegateValidator,
     EstimationsValidator,
     SingletonValidator,
+    LockingEventValidator,
     MessageValidator,
     ModuleTransactionValidator,
     MultisigTransactionValidator,
+    RankValidator,
```

Comment:
Related with https://github.com/safe-global/safe-client-gateway/pull/1229#discussion_r1510912118: I would also include the validators in the domain-specific module (i.e.: import them only if needed by the module).